Src/BootCamp.Chapter1/Program.cs

```diff
@@ -1,9 +1,14 @@
-﻿namespace BootCamp.Chapter1
+﻿
+
+namespace BootCamp.Chapter1
 {
     class Program
     {
         static void Main(string[] args)
         {
+            int[] test = new int[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4905609)